src/utils/string-utils.spec.js

```diff
@@ -49,4 +49,12 @@ describe('string-utils', () => {
     });
   });
 
+  describe('createShortHash', () => {
+    it('returns the same hash code for the same input', () => {
```

Comment:
I think this test can only be trusted if there's also a test that checks the negative scenario 😋

Reply:
The function and the test are gone now anyways :-)

Reply:
👍